Sources/Support/StoreMessagesHelper.swift

```diff
@@ -13,25 +13,37 @@
 
 import StoreKit
 
-class StoreMessagesHelper {
+protocol StoreMessagesHelperType {
```

Comment:
Extracted a `protocol` for this so we don't do `@unchecked Sendable` hacks.

Reply:
Could you elaborate here and explain why and how this works? Feel like the workaround will be hard to understand for anyone unfamiliar with Sendable. 

Might even be valuable to make it a comment

Reply:
non-final classes can't be Sendable. We use inheritance for mocking, which eliminates the ability to get automatic Sendable conformance. By using a protocol for mocking that's no longer a factor.